README.md

```diff
@@ -1,18 +1,23 @@
 <!--
-SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR), Forschungszentrum Jülich, Helmholtz-Zentrum Dresden-Rossendorf
+SPDX-FileCopyrightText: 2023 German Aerospace Center (DLR), \
+                             Forschungszentrum Jülich, \
+                             Helmholtz-Zentrum Dresden-Rossendorf
 
 SPDX-License-Identifier: CC0-1.0
 -->
 
 # HERMES continuous integration templates
 
-This repository contains templates for continuous integration systems that can be used to publish software automatically.
+This repository contains templates for continuous integration systems
+that can be used to publish software automatically.
 
 ## License
 
-- Continuous integration templates are licensed under [CC0-1.0](https://github.com/hermes-hmc/ci-templates/blob/main/LICENSES/CC0-1.0.txt).
+- Continuous integration templates are licensed under
+  [CC0-1.0](https://github.com/hermes-hmc/ci-templates/blob/main/LICENSES/CC0-1.0.txt).
```

Comment:
**though:** We should think about the actual refernece for this link. Looks kind of shaky for me....

Reply:
The same here. I tried to make it readable as text and also pass the typical rules. The original line was about 139 characters and this is too much.

Reply:
I totally agree with the line length here. However, I do not like the reference of the link too much...